platform/viewer/src/appExtensions/MeasurementsPanel/index.js

```diff
@@ -1,7 +1,15 @@
 import ConnectedMeasurementTable from './ConnectedMeasurementTable.js';
+import init from './init.js';
```

Comment:
Food for thought: Would the measurements table make more sense as part of the cornerstone extention? 🤔

Reply:
Great question. I can see the pros and cons. As extensions work today, I prefer them separate. It would also be nice if the measurements panel could show vtk measurements. Kind of need an `@ohif/core` measurements service with providers/consumers to make that work though.